metabase/metabase
# Start Cypress browser with prefers-reduced-motion set to true

One step to serve issue #17726 

Programatically sets this preference so that Cypress browser will honor it in e2e test environments.

In later PRs, as we remove or simplify animations whenever `prefers-reduced-motion` is `true`, e2e tests will reflect the changes.

## How to Test

Run Cypress locally in `open` mode.

Choose one or few tests to run in the browser.

1. Developer tools should open automatically just as they have been doing (this is another config that was already present in the same callback function, see diff in this PR)

2. Run the following line in the Cypress browser dev tools console:

```js
window.matchMedia('(prefers-reduced-motion: reduce)').matches;
```

console should output `true`.
